device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsIotHubConnectionTest.java

```diff
@@ -478,7 +478,7 @@ public void openThrowsIfProtonReactorThrows() throws TransportException
 
     // Tests_SRS_AMQPSIOTHUBCONNECTION_15_009: [The function shall trigger the Reactor (Proton) to begin running.]
     @Test
-    public void openTriggersProtonReactor(@Mocked final Reactor mockedReactor) throws TransportException, InterruptedException
+    public void openTriggersProtonReactorFinal(@Mocked final Reactor mockedReactor) throws TransportException, InterruptedException
```

Comment:
Final? I thought you removing all copies without Final, and renaming the *Final ones to the base name.

Reply:
Looks like this was just a find + replace error. I'll revert it. This test has nothing to do with the getters that I'm collapsing